src/core/stream.js

```diff
@@ -1858,6 +1859,10 @@ var CCITTFaxStream = (function CCITTFaxStreamClosure() {
     var refPos, blackPixels, bits, i;
 
     if (this.outputBits === 0) {
+      if (this.rowsDone) {
+        this.eof = true;
+        return null;
```

Comment:
Nit: Given the condition on the very next line, do we really need to `return null;` here as well?